cmd/minishift/cmd/artifact_preflight.go

```diff
@@ -39,21 +46,40 @@ type artifactCheckFunc func() bool
 // the returned status in a standardized way. If the test fails and returns a
 // false, the application will exit with errorMessage to describe what the
 // cause is.
-func artifactsCheckSucceedsOrFails(execute artifactCheckFunc, message string, errorMessage string) {
+func ocFlagCheckSucceedsOrFails(configNameOverrideIfSkipped string, execute artifactCheckFunc,
```

Comment:
It would now be a duplicate function of what already exists in preflight.go. see no additional value in having two functions doing the same. Since they are in the same package, no reason not to use it.

Reply:
+1 for preflight_artifacts.go